src/main/java/vg/civcraft/mc/civmodcore/playersettings/gui/ConfigSetAnyCommand.java

```diff
@@ -47,16 +45,7 @@ public boolean execute(CommandSender sender, String[] args) {
 	}
 	
 	public static UUID resolveName(String name) {
-		try {
 			return UUID.fromString(name);
-		}
```

Comment:
I still want that functionality, being able to run commands by name is very useful. Don't care that it's deprecated.